aws/resource_aws_glue_trigger.go

```diff
@@ -76,8 +82,14 @@ func resourceAwsGlueTrigger() *schema.Resource {
 							Elem: &schema.Resource{
 								Schema: map[string]*schema.Schema{
 									"job_name": {
-										Type:     schema.TypeString,
-										Required: true,
+										Type:          schema.TypeString,
+										Optional:      true,
+										ConflictsWith: []string{"predicate.0.conditions.0.crawler_name"},
```

Comment:
Same here, regarding the `conditions` attribute (the `.0.` syntax will work with the first configuration block, but it will not work with subsequent configuration blocks). Please see this upstream enhancement request: https://github.com/hashicorp/terraform-plugin-sdk/issues/71